osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/firmware/ImageTransfer.java

```diff
@@ -503,7 +503,7 @@ private boolean isSignature(final byte[] signature) {
     return true;
   }
 
-  static class ImageTranferProperties {
+  public static class ImageTranferProperties {
```

Comment:
The name looks like it has a pre-existing spelling mistake in it. Renaming it to `ImageTransferProperties` would get rid of this.

Reply:
renamed